users-status/README.md

```diff
@@ -194,23 +197,66 @@ Batch Updates the UserIds of the passed user list to Idle if the user is not OOO
 - **Body**
 
   - Attributes:
-  - users[] (required) : Specifies the user list of User ids to be updated.
+    - users (required, array): An array of objects, each representing a user with the following properties:
+      - userId (required, string): Specifies the unique identifier for the user.
+      - expectedState (required, string): Specifies the user's state, which must be one of the following values: IDLE or ACTIVE.
+  - Example:
+
+```
+  {
+      "users": [
+          {
+              "userId": "4kAkRv9TBlOfR6WEUhoQ",
+              "expectedState": "IDLE"
+          },
+          {
+              "userId": "SooJK37gzjIZfFNH0tlL",
+              "expectedState": "ACTIVE"
+          }
+      ]
+  }
+```
 
 - **Success Response:**
 
-  - **Code** 200</br>
+  - **Code**
+    - 200</br>
   - **Content**</br>
-    ```json
+
+```
     {
       "message": "String",
       "data": {
         "totalUsers": "Number",
-        "usersWithStatusUpdated": "Number",
-        "usersOnboardingOrAlreadyIdle": "Number"
+        "totalUnprocessedUsers": "Number",
+        "totalOnboardingUsersAltered": "Number",
+        "totalOnboardingUsersUnAltered": "Number",
+        "totalActiveUsersAltered": "Number",
+        "totalActiveUsersUnAltered": "Number",
+        "totalIdleUsersAltered": "Number",
+        "totalIdleUsersUnAltered": "Number"
       }
     }
 ```
 
+- **Example**
+
+```
+  {
+    "message": "users status updated successfully.",
+    "data": {
+        "totalUsers": 7,
+        "totalUnprocessedUsers": 1,
+        "totalOnboardingUsersAltered": 1,
```

Comment:
what is an onboardingUser ?

Reply:
users when they join rds they are assigned a state onboarding